src/interface/graph.thrift

```diff
@@ -90,6 +90,8 @@ struct ExecutionResponse {
     3: optional string error_msg;
     4: optional list<binary> column_names;  // Column names
     5: optional list<RowValue> rows;
+    6: optional list<binary> schema_prop_names;
```

Comment:
What's the purpose?

Reply:
Here line 6 and line 7 use to the relevant properties when show create tag/edge

Reply:
Could we define a independent structure for show create xxx.   
 It should not be outputted in table format.

Reply:
Yeah, you are right.  👍

Reply:
Good catch. 👍

Reply:
I will update it in `PR #439 Support show create tag/edge xxx, show create space xxx SQL `
This PR does not  contain` show create tag/edge xxx, show create space xxx SQL`. 
Please accept it